src/Apps/Article/Components/Sections/ArticleSectionImageCollection/ArticleSectionImageCollectionImage.tsx

```diff
@@ -38,14 +39,32 @@ const ArticleSectionImageCollectionImage: FC<
         position="relative"
       >
         <ArticleZoomButton width="100%" height="100%" onClick={handleClick}>
-          <Image
-            src={img.src}
-            srcSet={img.srcSet}
-            width="100%"
-            height="100%"
-            alt={figure.formattedMetadata ?? ""}
-            lazyLoad
-          />
+          {isFirst ? (
+            <img
+              src={img.src}
+              srcSet={img.srcSet}
+              width="100%"
+              height="100%"
+              alt={figure.formattedMetadata ?? ""}
+              fetchPriority="high"
+              decoding="async"
+              style={{
+                display: "block",
+                width: "100%",
+                height: "100%",
+                objectFit: "cover",
+              }}
+            />
+          ) : (
+            <Image
+              src={img.src}
+              srcSet={img.srcSet}
+              width="100%"
+              height="100%"
+              alt={figure.formattedMetadata ?? ""}
+              lazyLoad
+            />
+          )}
```

Comment:
I'm rendering this conditionally because I noticed that `<Image>` was  not allowing fetchPriority to actually be reflected here for some reason. Maybe it's a palette thing. So in the event that the image is the LCP element that we do want to be fetched first, I'm rendering a regular `<img>` tag, otherwise lazy loading is fine so I'm rendering the original `<Image>` from palette to maintain consistency.

Reply:
Though it *should* work, it's perhaps getting split on to the wrong element. On second thought, there's not a ton of benefit to using the `Image` component really if you're not using lazy loading or a blurhash, so this is fine too.